scripts/coverage.sh

```diff
@@ -25,5 +27,10 @@ cov ext/opentelemetry-ext-wsgi
 cov ext/opentelemetry-ext-zipkin
 cov docs/examples/opentelemetry-example-app
 
-coverage report
+# aiohttp is only supported on Python 3.5+.
```

Comment:
Just want to add a reference to https://github.com/open-telemetry/opentelemetry-python/issues/25 here, one more reason to drop 3.4 support.

Reply:
Any recommendations on how to exclude this package from testing on on 3.4? I started to get into the source code for running the tests pre-covid, but didn't get too far.